From 6cba36484bfbe41c27e2d07fd4e3065343661444 Mon Sep 17 00:00:00 2001
From: "yadi.hu@windriver.com" <yadi.hu@windriver.com>
Date: Thu, 13 Nov 2014 15:11:55 +0800
Subject: [PATCH 3/4] mpt2sas: Fix a warning '_ctl_mutex' defined but not used

linux/drivers/scsi/mpt2sas/mpt2sas_ctl.c:63:21: warning: '_ctl_mutex' defined but not used [-Wunused-variable]
 static DEFINE_MUTEX(_ctl_mutex);

Signed-off-by: Hu <yadi.hu@windriver.com>
---
 drivers/scsi/mpt2sas/mpt2sas_ctl.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/drivers/scsi/mpt2sas/mpt2sas_ctl.c b/drivers/scsi/mpt2sas/mpt2sas_ctl.c
index b7f887c..0bdada1 100644
--- a/drivers/scsi/mpt2sas/mpt2sas_ctl.c
+++ b/drivers/scsi/mpt2sas/mpt2sas_ctl.c
@@ -60,7 +60,6 @@
 #include "mpt2sas_base.h"
 #include "mpt2sas_ctl.h"
 
-static DEFINE_MUTEX(_ctl_mutex);
 static struct fasync_struct *async_queue;
 static DECLARE_WAIT_QUEUE_HEAD(ctl_poll_wait);
 
-- 
1.7.5.4

